app/app.go

```diff
@@ -368,31 +378,33 @@ func init() {
 
 // NewDesmosApp is a constructor function for DesmosApp
 func NewDesmosApp(
-	logger log.Logger, db dbm.DB, traceStore io.Writer, loadLatest bool, skipUpgradeHeights map[int64]bool,
-	homePath string, invCheckPeriod uint, encodingConfig simappparams.EncodingConfig,
-	appOpts servertypes.AppOptions, wasmEnabledProposals []wasm.ProposalType, baseAppOptions ...func(*baseapp.BaseApp),
+	logger log.Logger, db dbm.DB, traceStore io.Writer, loadLatest bool,
+	appOpts servertypes.AppOptions, wasmEnabledProposals []wasm.ProposalType,
+	baseAppOptions ...func(*baseapp.BaseApp),
```

Comment:
skipUpgradeHeights, homePath, invCheckPeriod are included inside servertypes.AppOptions, and encoding config is not input now.
See:
https://github.com/cosmos/cosmos-sdk/blame/v0.47.0/CHANGELOG.md#L201
https://github.com/cosmos/cosmos-sdk/blame/v0.47.0/CHANGELOG.md#L204

https://github.com/cosmos/cosmos-sdk/blob/v0.47.0/UPGRADING.md#constructor